core/kazoo_media/src/kz_media_single_proxy.erl

```diff
@@ -73,7 +79,8 @@ handle(Req0, {Meta, Bin}) ->
     Url = kz_json:get_value(<<"url">>, Meta, <<>>),
 
     Req2 = case ContentType of
-               CT when CT =:= <<"audio/mpeg">> orelse CT =:= <<"audio/mp3">> ->
+               CT when CT =:= <<"audio/mpeg">> orelse
```

Comment:
@johnwnowlin i guess you need to move the whole orelse clause to new line

` CT when CT =:= <<"audio/mpeg">> 
         orelse CT =:= <<"audio/mp3">> ->`